crates/bitwarden/src/auth/login/mod.rs

```diff
@@ -0,0 +1,55 @@
+#[cfg(feature = "internal")]
+use {
+    crate::{
+        client::{auth_settings::AuthSettings, Client},
+        error::Result,
+    },
+    bitwarden_api_identity::{
+        apis::accounts_api::accounts_prelogin_post,
+        models::{PreloginRequestModel, PreloginResponseModel},
+    },
+};
+
+pub mod response;
+
+mod password;
+pub use password::PasswordLoginResponse;
```

Comment:
Shouldn't these two also be behind the `internal` feature?

Reply:
Ideally, but for some reason the `PasswordLoginResponse` is used to unparse the other responses. I think the response parsing could use some more work but this PR is a good step in the right direction.